src/commands/optimize.ts

```diff
@@ -113,7 +113,7 @@ export default class Optimize extends Command {
         this.log(`Created file ${newPath}...`);
       } else if (this.outputMethod === Outputs.OVERWRITE) {
         await writeFile(specPath ?? 'asyncapi.yaml', optimizedDocument, { encoding: 'utf8' });
-        this.log(`Created file ${newPath}...`);
+        this.log(`Created file ${specPath ?? 'asyncapi.yaml'}...`);
```

Comment:
@rjambrecic afaik we should also change the log message to say `Update file` instead of `Created file`?

also could you introduce a new variable so we do not duplicate `specPath ?? 'asyncapi.yaml'` that is already in line 115?